docs/reference/indices/open-close.asciidoc

```diff
@@ -2,19 +2,42 @@
 == Open / Close Index API
 
 The open and close index APIs allow to close an index, and later on
-opening it. A closed index has almost no overhead on the cluster (except
-for maintaining its metadata), and is blocked for read/write operations.
-A closed index can be opened which will then go through the normal
-recovery process.
+opening it.
+
+A closed index is blocked for read/write operations and does not allow
+all operations that opened indices allow. It is not possible to index
+documents or to search for documents in a closed index. This allows
+closed indices to not have to maintain internal data structures for
+indexing or searching documents, resulting in a smaller overhead on
+the cluster.
+
+When opening or closing an index, the master is responsible for
+restarting the index shards to reflect the new state of the index.
+The shards will then go through the normal recovery process. The
+data of opened/closed indices is automatically replicated by the
+cluster to ensure that enough shard copies are safely kept around
+at all times.
 
 The REST endpoint is `/{index}/_close` and `/{index}/_open`. For
 example:
 
 [source,js]
 --------------------------------------------------
 POST /my_index/_close
+{
```

Comment:
this will look like this is the body of the request, whereas you would like to show this as response.

Reply:
Looks like I didn't write some docs for a long time... Thanks